services/users.js

```diff
@@ -73,7 +87,7 @@ class Service {
         attributes: { exclude: ['password'] },
         where: { id },
       });
-    return user || null;
+    return user?.dataValues ? user.dataValues : null;
```

Comment:
You use` .dataValues` in too many places, maybe we can have a function to extract that?

Reply:
I've already removed the `.dataValues` from all of the places in the latest commit to main, because it turns out it was redundant, as `user` object already has getters for columns